npm/webpack-dev-server-fresh/src/CypressCTWebpackPlugin.ts

```diff
@@ -0,0 +1,159 @@
+import type { Compiler, Compilation } from 'webpack'
```

Comment:
I'm actually not sure if this will track properly, since the files are being added/removed in different commits. But honestly now that we have this, working & tested, I'm wondering if it'd be much work to just replace the `webpack-dev-server` with the new implementation & provide a shim for the existing API use. We can discuss when we meet later.

Reply:
Nevermind, in talking w/ Zach think it should probably track fine.